api_db.php

```diff
@@ -404,10 +404,17 @@ function parseDBData($results, $interval, $from, $until) {
 				if($max === null || $max < $row[0])
 					$max = $row[0];
 
-				// Get the non-zero graph data
+				// $data[timestamp] = value_in_this_interval
 				$data[$row[0]] = intval($row[1]);
 			}
 
+			// Fall back to $from and $until if we read
+			// no data in the fetchArray while loop above
+			if($min === null)
+				$min = $from;
+			if($max === null)
+				$max = $until;
```

Comment:
We actually don't need the `$min` and `$max` variables, as the following inequality always holds due to the database `WHERE` clause:
```
$from <= $min <= $max <= $until
```